### ISSUE TITLE
Volunteer able to download copy of the Volunteer Agreements, dated with date downloaded

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/MyopicVicar#1152]
'Dated with date sigdownloaded' is a 'nice to have' but not essential.

**Comment by PatReynolds** [comment]
This has not been done - timestamping of agreement not yet in place.

**Comment by Captainkirkdawson** [comment]
Story spe was "'Dated with date sigdownloaded' is a 'nice to have' but not essential.". The volunteer can download the agreement and the date of the download is included. Not sure how "timestamping of agreement not yet in place."  is not done, Does not appear as a requirement. Mind you I do not understand what is implied by "timestamping of agreement"

**Comment by PatReynolds** [comment]
'timestapming of agreement' is the same as 'adding a date to the agreement on the date it is downloaded'.  Still a 'nice to have' but not (yet done).

**Comment by Captainkirkdawson** [comment]
Ah; very different requirement. Requirement is that the date (time?)  the TA is accepted is recorded for the userid.

**Comment by PatReynolds** [comment]
No, Kirk, that is a separate story.  But a good one.  This is just adding the date downloaded to the downloaded file.  Which is something we were asked for, and not essential.  It is probably worth asking the transcribers at the point where we get to considering this, whether or not it is still needed.

**Comment by PatReynolds** [comment]
As the original story needs attention on www.freeukgen.org.uk, not FreeREG or other sites, I have made a new story on the Coordination waffle board (#1009) to cover this.  

However, the story that got confused with this one, the need for us to record the date on which a transcriber signed the agreement, is wanted.

**Comment by Captainkirkdawson** [comment]
@PatReynolds  could we clarify what is required?. We seem to have moved away from the concept of signing towards the idea of acceptance of the agreement? Is there anything we need to do as a result?

**Comment by PatReynolds** [comment]
Accepting the Agreement = Signing, so no change.

**Comment by Captainkirkdawson** [comment]
@PatReynolds is there still a requirement to add the date of acceptance to the userid_detail record? 
We will not be able to do that for those that have already agreed.
